hubspot3/associations.py

```diff
@@ -0,0 +1,163 @@
+"""
```

Comment:
If that's something we added it may not be useful any more. See https://github.com/jpetrucciani/hubspot3/blob/master/hubspot3/crm_associations.py

Reply:
Thanks. It seems that it is the same thing for `pipelines`. I am looking at it